subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -949,6 +949,77 @@ class ArtifactTransformIntegrationTest extends AbstractHttpDependencyResolutionT
         output.contains("files: [lib.jar]")
     }
 
+    def "transforms registering the input as an output can use normalization"() {
+        file("input1.jar").text = "jar"
+        file("input2.jar").text = "jar"
+        buildFile("""
+            import org.gradle.api.artifacts.transform.TransformParameters
```

Comment:
Why is this single import required?

Reply:
I removed it, since the import is added in the test setup to the build script already. We need it, since our default imports don't work for inner classes, and we use `TransformationParameters.None` here.